src/docs/background/off-session-payments/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Automatic off-session payments in transaction process
 slug: off-session-payments-in-transaction-process
-updated: 2019-08-22
+updated: 2021-09-21
```

Comment:
Before merging, remember to update the `updated` attribute to the date when the change is deployed.